package.json

```diff
@@ -117,6 +117,7 @@
     "extends": "react-app"
   },
   "scripts": {
+    "prebuild": "git submodule update --init --recursive --remote && yarn lingui:compile",
```

Comment:
this and line 137 clash with @jesus-eff 's latest (merged) pr: https://github.com/OlympusDAO/olympus-frontend/pull/704. Just need to verify which is correct